addon/types/store.js

```diff
@@ -0,0 +1,22 @@
+class LocalStorageStore {
+  constructor(table, key) {
+    this.table = table;
```

Comment:
we really want to have the table here ?

Reply:
it's to have access to the columns "live-changes"

Reply:
Yes, but passing the `columns` to the `update` as argument could be more cleaner, no ?

Reply:
hmm.. you're right :) will update it